### ISSUE TITLE
Please handle the case your logits contain nans

### ISSUE THREAD

**Issue by ParisNeo** [turboderp/exllama#222]
![image](https://github.com/turboderp/exllama/assets/827993/29b43eec-bdb0-4511-8fbf-5f233ac5ee10)

Hi, I was prompting llama-2-7B and got into this error.

Can you please handle the case there are nans in the logits?

**Comment by turboderp** [comment]
NaN or infinities in the logits implies the model has failed for one reason or another. This may be caused by a bug in the implementation, or it may be a corrupted/incorrectly finetuned model, but I'd need a lot more details to start figuring out what the cause is.
